src/pages/game.tsx

```
@@ -0,0 +1,31 @@
+import Pile from "../shared/ui/Pile";
+import { randomCard } from "../shared/utils/randomCard";
+import { dragFunctions } from "../features/drag/dragFunctions";
+
+const Game = () => {
+  let deck = randomCard();
```

ESLint

[error] 'deck' is never reassigned. Use 'const' instead.

(prefer-const)